test/Tests/src/Semantic/Any_Spec.enso

```diff
@@ -42,19 +42,64 @@ spec =
             My_Type.Value 33 . x . should_equal "Any:(My_Type.Value 33)"
             With_X.Value 44 . x . should_equal "With_X:(With_X.Value 44)"
             With_Y.Value 44 . x . should_equal "With_Y:With_Y(44)"
-            Any.x . to_text . should_equal "Any.type.x[Any_Types.enso:6-26]"
             My_Type.x . should_equal "Any:My_Type"
             With_X.x . to_text . should_equal "Any:With_X"
-            With_X.y . to_text . should_equal "With_X.type.y[Any_Types.enso:12-32]"
+            Meta.is_a With_X.y Function . should_be_true
```

Comment:
This is a more persistent and maintainable way how to check if something is a `Function`. If you convert it to text via `to_text`, it will be different every time you fiddle with `Any_Types.enso` source, which is very inconvenient.